app/controllers/application_controller.rb

```diff
@@ -19,6 +19,15 @@ def after_sign_out_path_for(_)
     new_member_session_path
   end
 
+  # Pundit is going to call current_user in order to authorize whatever policy
+  # but we are using members and not users so we need to override the pundit_user
+  # Ref: https://github.com/varvet/pundit#customize-pundit-user
+  #
+  # @return [Member]
+  def pundit_user
```

Comment:
Don't know that it really matters for this, but when you name your user something else in devise, you still have those types of methods available to you, so in our case, current_member should work in controllers/views.